added git rules to readme (#27)
EDGENortheastern/sony-reddy-sys-test-dec-22: README.md

- [UAT](https://uat--loquacious-lolly-2c8d64.netlify.app/)

## Git Rules

>    Create a Git repository for every new project

Reviewer: ```suggestion
- Create a new Git repository for every new project
```

>    Create a new branch for every new feature

Reviewer: ```suggestion
- Create a **new ticket** for every **new feature** to be tested
```

>    Branch name should be of small alphabits and dash(-)

Reviewer: ```suggestion
- Add acceptance criteria to feature ticket
```

>    Use Pull requests to merge code to Master

Reviewer: ```suggestion
- Create a new branch for every new feature
- One feature - one ticket - one branch - one Pull Request - many commits
- A branch name should be of small alpha characters and dashes (-)
- Use Pull requests to merge code to uat and then uat to main in a release
```
